### DISCUSSION TITLE
Micronaut Ignite Development appetite

### DISCUSSION THREAD

**Discussion by sdelamo** [micronaut-projects/micronaut-ignite#153] [General]
Hello @schernolyas @pollend is see you contributed to this module. But this module was never released. Are you interested in contributing to it to get a release out ?

**Comment by pollend** [comment]
sorry, haven't look at this in such a long time. I haven't tracked what has change with Micronaut since working on this.  I think @dmagda was the main contact that was interested in having this support for Micronaut. I'm willing to put in some time if there is some strong interest in adding support for this.  If no one is using this module I don't think it would hurt to archive this repository.

**Comment by wetted** [comment]
I'd also like to know the plan for this repo. I am preparing other Micronaut repos (branches) for the next major Micronaut Framework release. Are we moving forward with Ignite, and if so which Micronaut version(s) do we want to target? 3, 4, both, none? I can also ignore it for now.
